custom_components/ecostream/climate.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import annotations
+
+import math
+from typing import Any, Optional
+
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.components.climate import (
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACAction,
+    HVACMode,
+)
+from homeassistant.const import UnitOfTemperature, PRECISION_WHOLE
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import EcostreamDataUpdateCoordinator, EcostreamWebsocketsAPI
+from .const import DOMAIN
+
+async def async_setup_entry(
+    hass: HomeAssistant, 
+    entry: ConfigEntry[EcostreamDataUpdateCoordinator], 
+    async_add_entities: AddEntitiesCallback,
+):
+    coordinator = entry.runtime_data
+
+    switches = [
+        EcostreamSummerComfortClimate(coordinator, entry),
+    ]
+
+    async_add_entities(switches, update_before_add=True)
+
+class EcostreamSummerComfortClimate(CoordinatorEntity, ClimateEntity):
+    _attr_hvac_modes = [HVACMode.OFF, HVACMode.COOL]
+    _attr_supported_features = ClimateEntityFeature.TARGET_TEMPERATURE
+
+    _attr_temperature_unit = UnitOfTemperature.CELSIUS
+    _attr_target_temperature_step = PRECISION_WHOLE
+    _attr_min_temp = 10
+    _attr_max_temp = 30
```

Comment:
This range is based on the range in the app. During my testing I found that higher and lower values are also accepted and persisted. However, I don't think we should allow that and the range is very sensible for the functionality as well.